File: result/translate/transcoder/java2python/python/PROGRAM_CENSOR_WORD_ASTERISKS_SENTENCE.py
def f_gold ( text , word ) :
    word_list = text.split ( 's+' )
    result = ''
    stars = ''
    for i in word :
        stars += '*'
    index = 0
    for i in word_list :
        if i == word :
            word_list [ index ] = stars
        index += 1
    for i in word_list :
        result += i + ' '
    return result

File: result/translate/chatgpt/python2java/java/PROGRAM_CENSOR_WORD_ASTERISKS_SENTENCE.java
public static String f_gold(String text, String word) {
    String[] wordList = text.split(" ");
    String result = "";
    String stars = "*".repeat(word.length());
    int count = 0;
    int index = 0;
    for (int i = 0; i < wordList.length; i++) {
        if (wordList[i].equals(word)) {
            wordList[index] = stars;
        }
        index++;
    }
    result = String.join(" ", wordList);
    return result;
}